### PR TITLE
Containerization / Docker

### PR DESCRIPTION
**Pull request by hilbert-erwin**

### CODE REVIEW

File: event-management-backend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+WORKDIR /build
+COPY pom.xml .
+RUN mvn dependency:go-offline
+
+COPY src/ /build/src/
+RUN mvn clean package -DskipTests spring-boot:repackage
+
+FROM openjdk:11-jre
+COPY --from=build /build/target/*.jar /app/*.jar
+CMD java -jar /app/*.jar
```
Comment (AndreiStefanie) [inline event-management-backend/Dockerfile]: There are 2 ways to run a command: shell (used here) and exec. The difference is that running the command in exec mode enables the app to listen to system signals (e.g. SIGTERM). More info [here](https://spring.io/guides/topicals/spring-boot-docker/) if you scroll below.

File: event-management-backend/src/main/resources/application.properties [hunk 2]
```diff
@@ -7,6 +7,6 @@ event-management.jwt.aws.region=eu-west-1
 server.error.include-message=always
 event-management.s3.tickets.bucketName=event-management-tickets
 
-spring.profiles.active=prod
+spring.profiles.active=${PROFILE:default}
```
Comment (AndreiStefanie) [inline event-management-backend/src/main/resources/application.properties:10]: That's an excellent point. Could we also use this approach for connecting to the database?

File: event-management-backend/Dockerfile (hunk 1 above)
Comment (hilbert-erwin) [reply event-management-backend/Dockerfile]: Changed.

File: event-management-backend/src/main/resources/application.properties (hunk 2 above)
Comment (hilbert-erwin) [reply event-management-backend/src/main/resources/application.properties:10]: Changing the profile already changed the db that was in use. I added env. variables for the db connection on top if this.

Comment (AndreiStefanie) [reply event-management-backend/src/main/resources/application.properties:10]: Excellent.

The issue was that the connection credentials were still baked into the application. If we change the DB, we have to rebuild the app. If we deploy it to multiple environments, we need multiple versions.

**Review by AndreiStefanie — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,14 @@
...
+The docker repo for the project is: [Dockerhub](https://hub.docker.com/repository/docker/erwinhilbert/event-management)
+
+(tag - version of the project)
+
+To download the image to your local pc: `docker pull erwinhilbert/event-management:tag`
+
+After making modifications to update the local image (curent dir. is backend): `docker build -t erwinhilbert/event-management:tag .`
+
+To run the project with h2 database: `docker run -it -p 8080:8080 erwinhilbert/event-management:tag`
+To run the project with RDS database: `docker run -it -p 8080:8080 --env PROFILE=prod erwinhilbert/event-management:tag`
```
Comment (AndreiStefanie) [inline README.md:13]: Could we also mention the other env variables?

Comment (hilbert-erwin) [reply README.md:13]: Changed the readme.

**Review by AndreiStefanie — APPROVED** [review]

**Review by SirVsbi — APPROVED** [review]
Looks good
Tested locally with both configurations and it's working fine! 👍

**Pull request merged**